app/server/utils/auth.py

```diff
@@ -95,28 +128,6 @@ def wrapper(*args, **kwargs):
                         }
                         return make_response(jsonify(response_object)), 401
 
-                    g.user = user
```

Comment:
shifted this stuff to post auth, because setting things pre-auth makes me uneasy.